### PR TITLE
Bug 1180130 - [Stingray] Scaffolding Device Deck

### PR DESCRIPTION
**Pull request by dwi2**
https://bugzilla.mozilla.org/show_bug.cgi?id=1180130

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8d58377f759969964e51c65b7c649f2dfc62d100

File: tv_apps/device-deck/locales/devices-deck.en-US.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+refresh=Refresh
+connected=Connected
+newly-found=Newly Found
```
Comment (rexboy7) [inline tv_apps/device-deck/locales/devices-deck.en-US.properties]: nit: new line

File: tv_apps/device-deck/bower_components/smart-button/style.css [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+  line-height: 8rem;
+  text-align: center;
+  color: #ffffff;
+  white-space: normal;
+  top: 0;
+  position: absolute;
+}
+
+smart-button[data-icon].focused:before {
+  color: #2d2d2d;
```
Comment (rexboy7) [inline tv_apps/device-deck/bower_components/smart-button/style.css]: This line turns the focused icon into gray color which does not match the visual spec. You may want to overriding it in `device.css`.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=495b4b75f1d2f9963665eabe129aab576e1dc720

**Pull request merged**